testing/internal/e2e/tests/static_with_vault/env_test.go

```diff
@@ -5,11 +5,13 @@ package static_with_vault_test
 
 import "github.com/kelseyhightower/envconfig"
 
+// `VaultAddr` is the address that the Boundary server uses to interact with the running Vault instance
```

Comment:
Put this over the field, not the struct. And no need for the backticks in code comments.